src/Command/Command.php

```diff
@@ -202,7 +201,12 @@ public function alterColumn(string $table, string $column, string $type): self
     public function batchInsert(string $table, array $columns, iterable $rows): self
     {
         $table = $this->queryBuilder()->quoter()->quoteSql($table);
-        $columns = array_map(fn (string $column) => $this->queryBuilder()->quoter()->quoteSql($column), $columns);
+
+        /** @psalm-var string[] $columns */
+        foreach ($columns as $i => $column) {
```

Comment:
foreach ($columns as &$column) {
    $column = $this->queryBuilder()->quoter()->quoteSql($column);
}
unset($column);